WolvenKit.App/ViewModels/GraphEditor/RedGraph.Scene.cs

```diff
@@ -370,22 +372,32 @@ public static RedGraph GenerateSceneGraph(string title, scnSceneResource sceneRe
 
                 foreach (var destination in sceneOutputConnector.Data.Destinations)
                 {
-                    var targetNode = nodeCache[destination.NodeId.Id];
-                    if (targetNode is IDynamicInputNode dynamicInputNode)
+                    try
                     {
-                        while (dynamicInputNode.Input.Count <= destination.IsockStamp.Ordinal)
+                        var targetNode = nodeCache[destination.NodeId.Id];
```

Comment:
Could you check if nodeCache has the key (maybe with TryGet) instead of the try/catch? It would be better to prevent the error alltogether rather than pushing it under the carpet

Reply:
it does work fine indeed. but there is another question. how to notify the user then?
![image](https://github.com/user-attachments/assets/6d441eee-5f60-4b3e-8e8c-2cb307b8ce8a)
im not sure its gonna work well in the game